specification/dimensions/invoicingentity.md

```diff
@@ -0,0 +1,32 @@
+# Provider
+
+An Invoicing Entity is an entity responsible for invoicing for the infrastructure and/or services consumed. It is commonly used for cost analysis and reporting scenarios.
+
+The InvoicingEntity column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values.
+
+See [Appendix: Origination of the cost data](../appendix/origination_of_cost_data.md) section for context (Provider/Publisher/Invoicing Entity) in various purchasing scenarios.
```

Comment:
We need to confirm how this link works in the spec PDF

Reply:
It won't. I added comment in the other PR for this. We need to use the anchor.

Reply:
changed to (#originationofcostdata)